### PR TITLE
Updates to Knative Client docs (Serverless)

### PR DESCRIPTION
**Pull request by abrennan89**
- Reviewing content, removing unnecessary `oc` includes and replacing these with `kn` specific instructions and xrefs/links as needed.
- Added "Before you begin" heading to call attention to the dependency on `oc` CLI and login
- Updated from 'Using Knative Client' to 'Getting started with Knative Client'

### CODE REVIEW

File: serverless/knative-client.adoc [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-Knative Client (`kn`) is the Knative command line interface (CLI). The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}. With `kn`, you can create applications and manage {product-title} projects from the terminal.
+Knative Client (`kn`) is the Knative command line interface (CLI).
+The CLI exposes commands for managing your applications, as well as lower level tools to interact with components of {product-title}.
+With `kn`, you can create applications and manage {product-title} projects from the terminal.
 
-Knative client does not have its own log in mechanism. To log in to the cluster you must install the `oc` CLI and use the `oc` login. Installation options for the CLI vary depending on your operating system.
+== Before you begin
 
+Knative Client does not have its own log in mechanism.
+To log in to the cluster you must install the `oc` CLI and use the `oc` login.
```
Comment (rhuss) [inline serverless/knative-client.adoc]:
```suggestion
To log in to the cluster you must install the `oc` CLI and use `oc login`.
```

**Review by rhuss — COMMENTED** [review]
looks good with one minor suggestion

**Comment by mgencur** [comment]
Looks good from QE perspective

File: modules/serverless-installing-cli-web-console.adoc [hunk 2]
```diff
@@ -0,0 +1,35 @@
+// Module is included in the following assemblies:
+//
+// serverless/knative-client.adoc
+
+[id="installing-cli-web-console_{context}"]
+= Installing the `kn` CLI using the {product-title} web console
+
+Once you have installed the {ServerlessOperatorName}, you will see a link to download the `kn` CLI for Linux, macOS and Windows from the *Command Line Tools* page in the {product-title} web console.
```
Comment (ahardin-rh) [inline modules/serverless-installing-cli-web-console.adoc]: Once the the {ServerlessOperatorName} is installed,

**Comment by ahardin-rh** [comment]
@abrennan89 Just one minor comment from me. Otherwise, looks good!

**Comment by mgencur** [comment]
LGTM

**Comment by vikram-redhat** [comment]
This should first go into master and then CPed

**Pull request merged**